### PR TITLE
deepanshu profile

### PR DESCRIPTION
**Pull request by deepanshu1334**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1741"></a>

### CODE REVIEW

File: public/data/deepanshu1334.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "url": "https://twitter.com/Deepans23454090",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Hashnode",
+        "url": "https://deepanshu669.hashnode.dev",
+        "icon": "hashnode"
+      },
+      {
+        "name": "Follow me on medium",
```
Comment (Panquesito7) [inline public/data/deepanshu1334.json:23]: I recommend capitalizing the `m` here. What do you think?
```suggestion
        "name": "Follow me on Medium",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant

File: public/data/deepanshu1334.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "url": "https://medium.com/@deepanshuchaudhry852",
+        "icon": "medium"
+      },
+      {
+        "name": "Follow me on linkedIn",
```
Comment (Panquesito7) [inline public/data/deepanshu1334.json:28]: I recommend capitalizing the `l` here. What do you think?
```suggestion
        "name": "Follow me on LinkedIn",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

**Comment by deepanshu1334** [comment]
kindly check these changes

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thank you! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**